cog/core/downtime.py

```
@@ -0,0 +1,97 @@
+
+
+def write_downtime_list(downtime_list: list[Downtime]):
+    with safe_open_w(DOWNTIME_PATH, encoding="utf-8") as file:
+        json.dump([downtime.to_dict() for downtime in downtime_list], file, indent=4)
+
+
+async def get_history(
+    bot: Bot, channel_id, *, after: datetime, before: datetime | None = None
+):
+    if before is None:
+        before = datetime.now()
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)